### PR TITLE
Update developers.html.md (#1)

### PR DESCRIPTION
**Pull request by ss22ever**
Fixes issue # (delete if not relevant)
Bug 1382317 of bugzilla@redhat.com
Changes proposed in this pull request:

Added dnf commands for Fedora 

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)
@sandrobonazzola
* Update developers.html.md

* Update developers.html.md

### CODE REVIEW

File: source/develop/developer-guide/vdsm/developers.html.md [hunk 1]
```diff
@@ -18,12 +18,24 @@ wiki_last_updated: 2015-08-26
 To build VDSM, enable the oVirt repositories by installing the ovirt-release rpm:
 
       yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
-
+      
+      
+ or if you are using Fedora 
+ 
+ 
+      dnf install http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
```
Comment (dankenigsberg) [inline source/develop/developer-guide/vdsm/developers.html.md]: This sounds a bit repetitive to me. How about asking to set `YUM=yum` or `YUM=dnf` depending on operating system, and then use `$YUM`?

Is it too technical for the casual admin?

Comment (ss22ever) [inline source/develop/developer-guide/vdsm/developers.html.md]: I think yes The casual admin may find it hard especially the new contributors .Else you decide?

Comment (ss22ever) [inline source/develop/developer-guide/vdsm/developers.html.md]: @dankenigsberg  If you want I can make it like this  : dnf install git (or yum install git on Fedora≤21 and CentOS)

**Comment by bproffitt** [comment]
@sandrobonazzola Did you review? Is it ready to review?

**Comment by sandrobonazzola** [comment]
Merging

**Pull request merged**